utils/rpms/daos.spec

```diff
@@ -216,8 +219,12 @@ scons %{?_smp_mflags}                 \
       USE_INSTALLED=all               \
       CONF_DIR=%{conf_dir}            \
       PREFIX=%{_prefix}               \
-      %{?scons_args}
+      %{?scons_args}                  \
+      %{?compiler_args}
```

Comment:
This is still not in the right place.  It should be in the `%build` section's `scons` command.